tendrl/node_agent/monitoring/collectd/templates/gluster/tendrl_glusterfs_brick_utilization.jinja

```diff
@@ -7,5 +7,6 @@
         integration_id "{{integration_id}}"
         graphite_host "{{ graphite_host }}"
         graphite_port "{{ graphite_port }}"
+        peer_name "{{ hostname }}"
```

Comment:
This suffices to use `hostname` for `peer_name` or we need some changes while setting up these templates on nodes from `tendrl-node-agent`?